sklearn/covariance/_graph_lasso.py

```diff
@@ -622,7 +660,8 @@ class GraphicalLassoCV(GraphicalLasso):
         If an integer is given, it fixes the number of points on the
         grids of alpha to be used. If a list is given, it gives the
         grid to be used. See the notes in the class docstring for
-        more details. Range is (0, inf] when floats given.
+        more details. Range is [1, inf) for an integer.
+        Range is (0, inf] for an array-like of floats.
```

Comment:
This is actually not tested, needs to be tested around here:

https://github.com/scikit-learn/scikit-learn/blob/47dd54f727f09a47e84fb4cb53d33f9b0f239edb/sklearn/covariance/_graph_lasso.py#L859-L861

Reply:
Added a test for this